### PR TITLE
Prepare upgraded Dodge for merge

### PR DESCRIPTION
**Pull request by zachlatta**

### CODE REVIEW

**Comment by zachlatta** [comment]
Don't merge yet, still making changes.

**Comment by zachlatta** [comment]
Alright, good to merge.

File: workshops/dodge/README.md [hunk 1]
```diff
@@ -178,7 +178,7 @@ Now that we've introduced motion into our game, it's time to put some restrictio
 
 ### Adding Controls
 
-We only want the player to move when we press the controls. p5.js gives us a way to detect when keys are being pressed, namely with the function `keyDown()`. This function takes an argument that represents a key. For our purposes, we'll be needing `LEFT_ARROW` and `RIGHT_ARROW`, which are [keyCodes](http://p5js.org/reference/#p5/keyCode) preset by p5.js to correspond to the left and right arrow keys, respectively.
+We only want the player to move when we press the controls. p5.js gives us a way to detect when keys are being pressed, namely with the function `keyDown()`. This function takes an argument that represents a key. For our purposes, we'll be needing `LEFT_ARROW` and `RIGHT_ARROW`, which are [`keyCodes`](http://p5js.org/reference/#p5/keyCode) preset by p5.js to correspond to the left and right arrow keys, respectively.
```
Comment (jkwok91) [inline workshops/dodge/README.md:181]: :+1:

File: workshops/dodge/README.md [hunk 2]
```diff
@@ -188,7 +188,7 @@ if (keyDown(RIGHT_ARROW)) {
 }
 ```
 
-And we'll have corresponding behavior assigned to the left arrow key, except that we'll decement the position by 1:
```
Comment (jkwok91) [inline workshops/dodge/README.md:191]: fail @ me

File: workshops/maze/README.md [hunk 3]
```diff
@@ -42,7 +42,7 @@ Let's understand the code in your working bin. There are two main parts:
 
 ### Adding the Maze Sprite
 
-Here is the top half of the code that is responsible for adding the maze sprite. If you're not quite sure what a sprite is, just [click here](../dodge/README.md#some-terminology) for the explanation in the [Dodge](../dodge/README.md) workshop.
+Here is the top half of the code that is responsible for adding the maze sprite.
```
Comment (jkwok91) [inline workshops/maze/README.md:45]: occam's razor :100:

**Pull request merged**